src/profiler/profiler.h

```diff
@@ -1193,7 +1213,11 @@ struct ProfileOperator : public ProfileEvent {
       if (attributes) {
         name_.append(attributes->to_string().c_str());
       }
-      categories_.set("operator");
+      if (std::string(name).find("::") != std::string::npos) {
```

Comment:
It would be great if you could turn this simple logic into a function with the explanation you have in this conversation, such as bool IsSubCustomOperator(const char *). Otherwise, I am afraid it is going to be hard to comprehend for future developers.

Reply:
Please replace this line with the function you added.

Reply:
fixed